db/seeds/source_types.yml

```diff
@@ -113,6 +114,7 @@ ansible-tower:
 azure:
   :product_name: Microsoft Azure
   :vendor: Azure
+  :icon_url: "/apps/chrome/assets/images/partners-icons/miscrosoft-azure.svg"
```

Comment:
typo?

Reply:
Yep that gotta be fixed in chrome though. I will take care of that, but it will take a while.

Reply:
:+1: ok

Reply:
Pr is here: https://github.com/RedHatInsights/insights-chrome/pull/547
It will take time though until a new version of chrome is released